### PR TITLE
open f32, f16 xdlops unit test

### PR DESCRIPTION
**Pull request by asleepzzz**
miopen driver can pass these tests  now,
so open again

### CODE REVIEW

**Review by whchung — APPROVED** [review]
LGTM. Let's see what CI shows.

**Review by whchung — APPROVED** [review]

File: mlir/utils/jenkins/Jenkinsfile.downstream [hunk 1]
```diff
@@ -28,8 +30,9 @@ pipeline {
                         -DBUILD_SHARED_LIBS=ON \
                         -DLLVM_BUILD_LLVM_DYLIB=ON \
                         -DMLIR_ROCM_RUNNER_ENABLED=1 \
-                        -DMLIR_MIOPEN_DRIVER_ENABLED=1 \
-                        -DMLIR_MIOPEN_SQLITE_ENABLED=1
+                        -DMLIR_MIOPEN_DRIVER_ENABLED=1
+                        # if we open sqlite, step4 lowering failed until adding the custom db
+                        # -DMLIR_MIOPEN_SQLITE_ENABLED=1
```
Comment (jerryyin) [inline mlir/utils/jenkins/Jenkinsfile.downstream:35]: @asleepzzz I don't quite get why sqlite db need to be turned off in CI?

File: mlir/utils/jenkins/Jenkinsfile.downstream [hunk 2]
```diff
@@ -18,6 +18,8 @@ pipeline {
                 checkout scm
                 sh '''
                     mkdir -p build && cd build
+                    # Set the path to llvm-symbolizer
+                    PATH=$PATH:/opt/rocm-4.0.0/llvm/bin/
```
Comment (jerryyin) [inline mlir/utils/jenkins/Jenkinsfile.downstream:22]: @asleepzzz Why do we need this? Why does this has to be done before the build happens? How is `llvm-symbolizer` used in our build process and how does it fail the build process if this is not done?

File: mlir/utils/jenkins/Jenkinsfile.downstream (hunk 1 above)
Comment (jerryyin) [reply mlir/utils/jenkins/Jenkinsfile.downstream:35]: The dockerfile from our channel make sure it always have the sqlite db installed. Removing those lines will reduce our test coverage. Refer to below:

https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/blob/8e1e2686e6f69874fdf5a5955253ddc8f1e1f082/mlir/utils/jenkins/Dockerfile.rocm#L95-L98

**Pull request merged**